docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -13,18 +13,23 @@ Synopsis
 Description
 -----------
 
-Returns approximated statistics for the named table or for the results of a query.
+Returns approximated statistics for the named table or for the results of a
+query. Returns ``NULL`` for any statistics that are not populated or
+unavailable on the data source.
 
-Statistics are returned for each column, along with a summary row.
+Statistics are returned as a row for each table's column, plus a summary row for
+each table. The following table lists the returned columns and what statistics.
+they represent. Any additional statistics collected on the data source, other
+than those listed here, are not included.
 
-==========================  =============================================================
-Column                      Description
-==========================  =============================================================
-``column_name``             The name of the column (``NULL`` for the summary row)
-``data_size``               The total size in bytes of all of the values in the column
-``distinct_values_count``   The number of distinct values in the column
+==========================  ============================================================= =================================
+Column                      Description                                                   Notes
+==========================  ============================================================= =================================
+``column_name``             The name of the column                                        Returned in column rows
```

Comment:
The notes column is inaccurate here. The column will *always* be returned, no matter what the row is depicting (same goes for line 32, etc.). The Note here should say "``NULL``" for summary row, as should lines 29, 30, 31, 33, 34. For line 32, should say "``NULL`` for column stat rows"

Reply:
Updated